<?xml version="1.0"?>
<launch>
    <param name="map_1" value="CMI_outerloop_mach_e_wider_lane.csv"/>
    <param name="map_2" value="CMI_outerloop_mach_e.csv"/>
    <param name="spd_map_1" value="ITIC_Lane_Change_Modeling_StartLane.csv"/>
    <param name="spd_map_2" value="ITIC_Lane_Change_Modeling_EndLane.csv"/>
    <param name="track_style" value="Rally"/>
    <param name="num_vehicles" value="10"/>
    <node pkg="cra_traffic_sim" type="urban_roadside_scenario.py" name="CRA_urban_roadside_traffic_scene" output="screen"/>
</launch>